src/explore-education-statistics-admin/src/components/editable/__tests__/ReorderableAccordion.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import noop from 'lodash/noop';
+import Button from '@common/components/Button';
+import ReorderableAccordion from '../ReorderableAccordion';
+
+describe('ReorderableAccordion', () => {
+  test("it doesn't show controls if `canUpdateRelease` is false", () => {
```

Comment:
Bit of nitpick, but don't need to write 'it' at the start of test cases as it's a bit of unnecessary filler.

Same with the other test case.